capa/render/proto/capa.proto

```diff
@@ -63,75 +62,86 @@ message BasicBlockLayout {
 message BytesFeature {
   string type = 1;
   string bytes = 2;
-  string description = 3;
+  optional string description = 3;
 }
 
 message CharacteristicFeature {
   string type = 1;
   string characteristic = 2;
-  string description = 3;
+  optional string description = 3;
 }
 
 message ClassFeature {
   string type = 1;
```

Comment:
i wonder if we should be worrying about `type` being a reserved keyword in some languages?

im leaning towards "no" and only focusing on Python, since thats what capa is written in.